web_src/js/features/repo-home.js

```diff
@@ -152,12 +155,14 @@ export function initRepoTopicBar() {
   });
 
   $.fn.form.settings.rules.validateTopic = function (_values, regExp) {
-    const $topics = $topicDropdown.children('a.ui.label');
-    const status = !$topics.length || $topics.last()[0].getAttribute('data-value').match(regExp);
+    const topics = topicDropdown.querySelectorAll(':scope > a.ui.label');
+    const lastTopic = topics[topics.length - 1];
+    const status = !topics.length || lastTopic.getAttribute('data-value').match(regExp);
     if (!status) {
-      $topics.last().removeClass('green').addClass('red');
+      lastTopic.classList.remove('green');
+      lastTopic.classList.add('red');
```

Comment:
It doesn't seem right. If `!status`, it might also means "no last topic"

Reply:
https://github.com/go-gitea/gitea/pull/30191/commits/76d91b0aa122d6c1b49f54f1323128f26de817db